library/include/hipsparse.h

```diff
@@ -2318,7 +2426,7 @@ hipsparseStatus_t hipsparseSbsrilu02_analysis(hipsparseHandle_t         handle,
                                               int                       mb,
                                               int                       nnzb,
                                               const hipsparseMatDescr_t descrA,
-                                              const float*              bsrSortedValA,
+                                              float*                    bsrSortedValA,
```

Comment:
Note: cusparse does not take a const float*. Therefore hipsparseSbsrilu02 api cannot use a const float* here since we cant cast const float* to float* when using cuda backend.